test/contrib/rails/errors_test.rb

```diff
@@ -52,6 +52,36 @@ class TracingControllerTest < ActionController::TestCase
     assert_nil(span.get_tag('error.msg'))
   end
 
+  test 'missing rendering should close the template Span' do
```

Comment:
if we disable the `@tracer.writer.spans` reset after each test (like we're doing in real code with our Rack middleware), this simple test reproduces a memory leak.

Reply:
👍